cli/sawtooth_cli/config.py

```diff
@@ -53,6 +53,34 @@ def add_config_parser(subparsers, parent_parser):
                                            dest="subcommand")
     config_parsers.required = True
 
+    # The following parser is for the `genesis` subcommand.
+    # This command creates a batch that contains all of the initial
+    # transactions for on-chain settings
+    genesis_parser = config_parsers.add_parser('genesis')
+    genesis_parser.add_argument(
+        '-k', '--key',
+        type=str,
+        help='the signing key for the resulting batches '
+             'and inital authorized key')
+
+    genesis_parser.add_argument(
+        '-o', '--output',
+        type=str,
+        default='config-genesis.batch',
+        help='the name of the file to ouput the resulting batches')
+
+    genesis_parser.add_argument(
+        '--approval_threshold',
```

Comment:
Use - instead of _ in the command line arguments.

This one is quite long.  If a shorter argument (--threshold?) won't work, maybe add a short option (-T?).